### PR TITLE
Fix the inconsistent spacing between navbar items

### PR DESCRIPTION
**Pull request by anders-schneider**
Before:

image

After:

image

But on mobile (or for narrow window widths), it will still do inconsistent spacing so that each navbar item gets space according to how wide the text is:

image

AFAICT our codebase only uses the `ProgressNav` for the eligibility questionnaire, so the entirety of this change should be displayed in those screenshots above.

### CODE REVIEW

File: sites/public/styles/overrides.scss [hunk 1]
```diff
@@ -31,6 +31,10 @@ h1.form-card__header_title {
   text-transform: capitalize;
 }
 
+li.progress-nav__item {
```
Comment (willrlin) [inline sites/public/styles/overrides.scss:34]: Please add a comment explaining why this is the case.

**Review by willrlin — COMMENTED** [review]
Maybe you could start a wiki page called like Future Gotchas or something and mention that the nav item is restricted to 20% width for the questionnaire but if used for something else, we'd have to change the styling.

File: sites/public/styles/overrides.scss (hunk 1 above)
Comment (anders-schneider) [reply sites/public/styles/overrides.scss:37]: Done, thanks!

**Comment by anders-schneider** [comment]
> Maybe you could start a wiki page called like Future Gotchas or something and mention that the nav item is restricted to 20% width for the questionnaire but if used for something else, we'd have to change the styling.

I found a "Gotchas" section in our existing wiki (https://github.com/CityOfDetroit/bloom/wiki/Troubleshooting#gotchas) and added it there!

**Review by tomgebauer — APPROVED** [review]
LGTM! Good eye!

**Pull request merged**